src/Dfc.CourseDirectory.Database/Pttcd/StoredProcedures/RefreshFindACourseIndex.sql

```diff
@@ -0,0 +1,160 @@
+﻿CREATE PROCEDURE [Pttcd].[RefreshFindACourseIndex]
+	@CourseRunIds [Pttcd].[GuidIdTable] READONLY
+AS
+BEGIN
+
+	WITH RecordsCte AS (
+		SELECT
+			-- Id is the CourseRun's ID appended with the Region, if there is one
+			CONVERT(VARCHAR(36), cr.CourseRunId) + CASE WHEN crr.RegionId IS NOT NULL THEN '-' + crr.RegionId ELSE '' END AS Id,
```

Comment:
guess we can use that concat_ws thing again, just for kicks, might be a little more readable